scripts/github_issue_retriever.py

```diff
@@ -48,14 +50,20 @@ def replace_with_private(
                     continue
 
                 owner, repo, issue_number = extracted
-                private_issue = self.github.fetch_issue_by_number(
-                    owner, repo, issue_number
-                )
-                if private_issue:
-                    item["title"] = private_issue["title"]
-                    item["body"] = private_issue["body"]
-                    updated_ids.add(item["id"])
-                    updated_issues.append(item)
+                try:
+                    private_issue = self.github.fetch_issue_by_number(
+                        owner, repo, issue_number
+                    )
+
+                    if private_issue:
+                        item["title"] = private_issue["title"]
+                        item["body"] = private_issue["body"]
+                        updated_ids.add(item["id"])
+                        updated_issues.append(item)
+
+                except requests.HTTPError as e:
+                    logger.info(e)
+                    pass
```

Comment:
Nit: `pass` is not needed here because you already have at least one statement in this block (`logger.info(e)`).